app/Formatter/BoardSwimlaneFormatter.php

```diff
@@ -104,19 +104,39 @@ public function format()
             }
         }
 
+        $this->addNumberOfOpenTasksInColumn();
+
         return $this->swimlanes;
     }
 
     /**
-     * Calculate stats for each column acrosss all swimlanes
+     * Add number of open tasks to each column, ignoring the current filter
+     *
+     * @access protected
+     */
+    protected function addNumberOfOpenTasksInColumn()
+    {
+        $columns = $this->columnModel->getAllWithTaskCount($this->swimlanes[0]['columns'][0]['project_id']);
+
+        if (!empty($columns)) {
+            foreach ($this->swimlanes as &$swimlane) {
+                foreach ($swimlane['columns'] as $columnIndex => &$column) {
+                    $column['column_nb_open_tasks'] = $columns[array_search($column['id'], array_column($columns, 'id'))]['nb_open_tasks'];
```

Comment:
Is it possible to move this line in the loop above (line 103)? That would avoid doing twice the same loop.